src/components/CreateContact/CreateContact.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react'
+import styled from 'styled-components/macro'
+
+export default function CreateContact( onSubmit ) {
```

Comment:
where is onSubmit comming from? Do you need it?

Reply:
OnSubmit is a leftover from old code. Thanks for the hint!